Automattic/newspack-newsletters
# Add another layout and add scroll effect to layout picker

### All Submissions:






### Changes proposed in this Pull Request:

Added an 8th layout plus a tiny scroll effect when selecting a layout

![](https://user-images.githubusercontent.com/177929/164709356-eac9ae82-c8b9-4855-8235-4b8264bbd563.gif)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Do you see the effect when hovering?
3. Select Layout 8 and publish/test

### Other information:
